clusterloader2/pkg/measurement/interface.go

```diff
@@ -37,7 +37,8 @@ type MeasurementConfig struct {
 	ClusterLoaderConfig *config.ClusterLoaderConfig
 
 	// Identifier identifies this instance of measurement.
-	Identifier    string
+	// It could be either of type string or map.
+	Identifier    interface{}
```

Comment:
Do we need this change? It looks like the identifier in the measurement_executor when calling mm.Execute will be always string, right?

Reply:
I think we need it, as it can be map[string]string.

That said - I don't like relying on interface{}.
Instead, what we should make is create an Identifier interface and implement necessary methods
(from the quick glance, probably String() and Kind() or sth like that).

Reply:
I don't see a place where it can be map[string]. It needs to be interface in clusterloader2/api/types.go. But then the measurement_executor always calls mm.Execute using a string identifier (lines 42 and 64). We're implementing the multi-measurement at the measurement api level, but we're not pushing it down to the measurement implementations. A single measurement will still operate on a single identifier / instance. Please correct me if I'm missing something.

Reply:
OK - I guess my comment was to the API part. I just don't want to have an arbitrary interface{} inside the API.

Reply:
yup that made sense.. i changed it's type back to string.